addons/sys_radio/stringtable.xml

```diff
@@ -65,5 +65,8 @@
             <Chinesesimp>无线电正在被另一名玩家进行设置。</Chinesesimp>
             <Korean>다른 플레이어가 무전기를 설정중입니다.</Korean>
         </Key>
+        <Key ID="STR_ACRE_sys_radio_radioJustOpened">
+            <English>Somebody beat you when trying to configure the radio and you respectfully step away.</English>
```

Comment:
Something shorter please. Eg. "Radio opened by someone else.".
I'd just use `alreadyOpenedRadio` tbh.

Reply:
There is a string with that name already. This is a special case where two players open simultaneously the radio. In this case, the radio will b e opened and closed briefly for the one that registers later on the server.

Reply:
So i will delete this one and use the generic one as well.

Reply:
Yeah, doesn't matter which case happens, it's already in use and should be descriptive enough.